docs/dev/table/catalog.md

```diff
@@ -189,11 +190,14 @@ The following limitations in Hive's data types impact the mapping between Flink
 
```

Comment:
should be "*minimum* length is 255". It was my mistake, but would be great to fix that too

Reply:
It' not a mistake according to [hive char type](https://cwiki.apache.org/confluence/display/Hive/LanguageManual+Types#LanguageManualTypes-char) definition.